README.md

```diff
@@ -107,6 +107,10 @@ The logics for credentials is by decreasing order of importance:
 4. Then constant login/password keys are read
 5. If still either "login" or "password" is not defined, the credentials are considered invalid and anonymous acces is used for the instance without authentication
 
+__Nota__:
+
+At each level where login/password are specified, one can add the option `verify: false` to deactivate verification of HTTPS certificates. By default the HTTPS certificate must be valid, which is equivalent to `verify: true`
```

Comment:
I do not see `verify` in `substitute_single_credentials_from`
So it seems not possible to define verify with credentials at common (parent) level.
Maybe I miss something.

I would add in doc that this should only be used for development purpose as credentials will we sent unencrypted on the network.